webapp/resources/src/main/webapp/javascript/eXo/UIUserProfilePopup.js

```diff
@@ -294,12 +294,17 @@
 
                  tdProfile.append(aProfile);
 
+                 if(json.position){
```

Comment:
Does this fix the issue as proposed in https://jira.exoplatform.org/browse/PLF-8528?focusedCommentId=511347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-511347

Reply:
Sorry for this late answer,the proposed changes indeed fix this issue as [proposed](https://jira.exoplatform.org/browse/PLF-8528?focusedCommentId=511347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-511347) as displayed in this [screenshot](https://jira.exoplatform.org/secure/attachment/122294/Screen%20Shot%202020-04-15%20at%209.36.24%20PM.png )

Reply:
Ok could you include the content of the first 
```
 if(json.position)
```
inside the later one

Reply:
> Ok could you include the content of the first inside the later one

Done